modules/coralogix-aws-shipper/main.tf

```diff
@@ -138,16 +146,8 @@ module "lambda" {
     secret_permission = local.secret_access_policy
     destination_on_failure_policy = local.destination_on_failure_policy
   }
-  # The condition will first check if the integration type is cloudwatch, in that case, it will
-  # Allow the trigger from the log groups otherwise it will check if sns in enabled in
-  # case that it's not then the trigger will be triggered from the bucket
-
-  allowed_triggers = var.integration_type == "CloudWatch" ? {
```

Comment:
I'm not too familiar with this TF or this integration, but do we lose the CloudWatch trigger configuration with this removal? Does this get resolved elsewhere, or is it no longer necessary?

Reply:
The trigger configuration is in the aws_cloudwatch_log_subscription_filter resource, this part of the code creates permission for the cloudwatch to trigger the lambda, i removed this part because there was an issue in the lambda module when I used it which caused an error if you tried to use log group with a dot in the name(it said that you can't use a dot in the name even though according to aws you can). That is why i ended up creating the aws_lambda_permission resource by myself.